README.md

```diff
@@ -43,7 +43,7 @@ As the controller is still under development, the helm chart is not available in
    ```shell
    VERSION=e2e make docker-build
```

Comment:
In this section you are explaining how to build the cluster and use the provider container but are building with the `e2e` image tag, this should either be `v0.0.1` here instead or we should also state that they should change the image tag in `secretsync/values.yaml` in the `1. Configure the provider container` section as well as this will fail otherwise.

Reply:
added image tag override to update version to e2e in helm install